src/content/core/namespaces.mdx

```diff
@@ -1,53 +1,58 @@
 ---
-title: Secure and isolated namespaces
-description: Create and manage secure and isolated Kubernetes namespaces for you and your team
+title: Secure and isolated Namespaces
+description: Create and manage secure and isolated Kubernetes Namespaces for you and your team
 sidebar_label: Namespaces
 id: namespaces
 ---
 
 import Image from "@theme/Image";
 
-With Okteto dashboard you can create and manage secure and isolated Kubernetes namespaces for you and your team.
 
-The Okteto platform will take care of all the required RBAC roles and bindings, resources quotas and limits, pod security policies, and network policies so everyone in your team can safely deploy and develop applications in Kubernetes.
+Namespaces in Okteto offer a flexible way to organize your Development Environments. A Namespace can last for a few weeks for developing a feature, or a couple days for a bug fix.
+Not only do Namespaces give you an individual separation of context while you are working, but you can also use shared Namespaces to collaborate with you team before anyone pushes to git.
 
-## Manage your namespaces
+In Okteto, Namespaces are categorized into two types:
 
-You can create up to five namespaces per account. Go to the Okteto dashboard and click on the (+) symbol on the left.
+- **Personal Namespace**: Automatically generated for each user, this Namespace is uniquely named after the user's username or email, ensuring a personalized environment. These cannot be deleted, but their resources can be destroyed allowing the Namespace to be reused.
+- **Non-Personal Namespaces**: These are additional Namespaces created by users or through automation, facilitating broader project collaboration and management. They may be deleted or transferred to other users.
+
+With the Okteto Dashboard you can create, launch, and manage secure and isolated Namespaces for you and your team.
```

Comment:
I think we should emphasize the self-service nature of the namespaces. "You and your team can create namespaces bla bla bla"